Cargo.toml

```diff
@@ -143,6 +143,7 @@ members = [
     # services
     "services/collab-history",
     "services/realtime",
+    "libs/client-api-for-wasm"
```

Comment:
How about naming the crate `client-api-wasm`? This aligns with the naming conventions of other crates.